apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PatientResourceProvider.java

```diff
@@ -454,7 +459,7 @@ private String partDFieldFor(CcwCodebookVariable month) {
     if (hasHICN(identifiers)) joinsClause += "left join fetch b.beneficiaryHistories ";
 
     String query =
-        "select b from Beneficiary b "
+        "select distinct b from Beneficiary b "
```

Comment:
Just a note: I haven't tested this at all, so please make sure you do.

Reply:
fix that

Reply:
?